decidim-core/lib/tasks/decidim_tasks.rake

```diff
@@ -15,6 +15,7 @@ namespace :decidim do
       decidim
       decidim_accountability
       decidim_admin
+      decidim_api
```

Comment:
Is this necessary? In #8631 we didn't add it, and it worked, right? Also, we would not need to "Install migrations from Decidim to the app." -- or maybe we should update that description?